lib/deploy/storage/deploy.js

```diff
@@ -0,0 +1,25 @@
+'use strict';
+
+var _ = require('lodash');
+var RSVP = require('rsvp');
+
+var gcp = require('../../gcp');
+var utils = require('../../utils');
+
+module.exports = function(context, options) {
+  var files = _.get(context, 'storage.rules');
+  if (!files) {
+    return RSVP.resolve();
```

Comment:
Perhaps add a message saying "No rules to deploy"?

Reply:
We don't want to do this, since someone running `firebase deploy` without having initialized their project for storage wouldn't have expected storage to deploy and the message would just be confusing.